scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -48,6 +48,13 @@ sealed trait Execution[+T] extends java.io.Serializable {
   import Execution.{ EvalCache, FlatMapped, GetCounters, ResetCounters, Mapped, OnComplete, RecoverWith, Zipped }
 
   /**
+   * Lift an Execution into a Try
+   * @return Execution with object wrapped in a Try
+   */
+  def liftToTry: Execution[Try[T]] =
+    map(e => Try(e)).recoverWith{ case throwable => Execution.from(Failure(throwable)) }
```

Comment:
Can't actually use Execution.failed here because it will fail the Execution itself when what we really want is the Failure object inside of a good Execution. 

Success was a good catch though and is fixed.